src/rgw/rgw_lc.cc

```diff
@@ -1661,6 +1735,15 @@ int RGWLC::bucket_lc_process(string& shard_id, LCWorker* worker,
     worker->workpool->drain();
   }
 
+  std::unique_ptr<rgw::sal::User> user;
```

Comment:
how is this change related to notifications?

Reply:
it is, or was, required to fetch the correct state into the sal object